app/src/main/res/layout/license_item.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
+
+    <import type="android.view.View" />
+
+    <variable
+      name="viewModel"
+      type="org.oppia.android.app.help.thirdparty.LicenseItemViewModel" />
+  </data>
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="@color/white">
+
+    <TextView
+      android:id="@+id/license_name_text_view"
+      style="@style/Subtitle1"
+      android:layout_width="match_parent"
+      android:background="@drawable/general_item_background_border"
+      android:fontFamily="sans-serif-smallcaps"
+      android:onClick="@{() -> viewModel.clickOnLicenseItem()}"
+      android:paddingStart="16dp"
+      android:paddingTop="20dp"
+      android:paddingEnd="16dp"
+      android:paddingBottom="20dp"
+      android:text="@{viewModel.licenseName}"
+      android:textColor="@color/black"
+      android:textSize="14sp"
```

Comment:
Use `Subtitle2` and remove `textSize`.

Also suggest removing `textColor=black` too.

Reply:
Done.